variables.tf

```diff
@@ -511,6 +511,12 @@ variable "resource_group_name" {
   description = "Name of pre-exising resource group. Leave blank to have one created"
 }
 
+variable "vnet_resource_group_name" {
+  type    = string
+  default = null
+  description = "Name of a pre-exising resource group containing the BYO vnet resource. Leave blank if you are not using a BYO vnet or if the BYO vnet is co-located with the SAS Viya4 AKS cluster."
```

Comment:
I would change the wording here. Remove the Leave blank wording. If it's not set it's ignored. Nothing more to say here.